plugins/Phaser/PhaserEffect.cpp

```diff
@@ -0,0 +1,613 @@
+		const float order = orderBuf ? orderBuf->value(f) : m_phaserControls.m_orderModel.value();
+		const bool enableLFO = enableLFOBuf ? enableLFOBuf->value(f) : m_phaserControls.m_enableLFOModel.value();
+		const float amount = amountBuf ? amountBuf->value(f) : m_phaserControls.m_amountModel.value();
+		const float inFollow = inFollowBuf ? inFollowBuf->value(f) : m_phaserControls.m_inFollowModel.value();
+		const bool invert = invertBuf ? invertBuf->value(f) : m_phaserControls.m_invertModel.value();
+		const bool wetIsolate = wetBuf ? wetBuf->value(f) : m_phaserControls.m_wetModel.value();
+		const int mode = modeBuf ? modeBuf->value(f) : m_phaserControls.m_modeModel.value();
+		const bool analog = analogBuf ? analogBuf->value(f) : m_phaserControls.m_analogModel.value();
+		const float analogDist = analogDistBuf ? analogDistBuf->value(f) : m_phaserControls.m_analogDistModel.value();
+		const bool doubleVal = doubleBuf ? doubleBuf->value(f) : m_phaserControls.m_doubleModel.value();
+		const bool alias = aliasBuf ? aliasBuf->value(f) : m_phaserControls.m_aliasModel.value();
+		const float delay = delayBuf ? delayBuf->value(f) : m_phaserControls.m_delayModel.value();
+		const float cutoffControl = cutoffControlBuf ? cutoffControlBuf->value(f) : m_phaserControls.m_cutoffControlModel.value();
+		const float delayControl = delayControlBuf ? delayControlBuf->value(f) : m_phaserControls.m_delayControlModel.value();
+		const float distortion = (distortionBuf ? distortionBuf->value(f) : m_phaserControls.m_distortionModel.value()) * 0.01f;
+		const float feedback = (feedbackBuf ? feedbackBuf->value(f) : m_phaserControls.m_feedbackModel.value()) * 0.01f;
+
+		// Calculate input follower values
+		const double sAbs[2] = {abs(s[0]), abs(s[1])};
+		for (int i = 0; i < 2; i++)
+		{
+			if (sAbs[i] > m_currentPeak[i])
+			{
+				m_currentPeak[i] = qMin(m_currentPeak[i] * m_attCoeff, sAbs[i]);
+			}
+			else
+			if (sAbs[i] < m_currentPeak[i])
+			{
+				m_currentPeak[i] = qMax(m_currentPeak[i] * m_relCoeff, sAbs[i]);
+			}
+
+			m_currentPeak[i] = qBound(PHA_NOISE_FLOOR, m_currentPeak[i], 10.0f);
+		}
+
+		// Calculate real allpass filter frequencies
+		float lfoResults[2] = {0, 0};
+		if (enableLFO)
+		{
+			m_lfo->tick(&lfoResults[0], &lfoResults[1]);
+		}
+
+		const float realLfo[2] = {lfoResults[0] * amount, lfoResults[1] * amount};
+		const float realInFollow[2] = {m_currentPeak[0] * inFollow, m_currentPeak[1] * inFollow};
+		for (int i = 0; i < 2; i++)
+		{
+			m_displayCutoff[i] = qBound(20.f, detuneWithOctaves(cutoff, realLfo[i] + realInFollow[i]), 20000.f);
+			m_realCutoff[i] = qBound(20.f, detuneWithOctaves(cutoff, (realLfo[i] + realInFollow[i]) * cutoffControl), 20000.f);
+		}
+		
+		/*
+		Precalculate the allpass filter coefficients.
+		Because of very aggressive CPU optimization, some of
+		the coefficient names are inaccurate concerning what
+		they actually do.
+		*/
+		float b0[2];
+		float b1[2];
+		for (int b = 0; b < 2; ++b)
```

Comment:
Using `a`, `b`, `c`  for loop variables is probably unexpected. Use `i`, `j`, `k` or something meaningful like `ch`.